app/uk/gov/hmrc/tai/service/JrsService.scala

```diff
@@ -39,6 +39,9 @@ class JrsService @Inject()(jrsConnector: JrsConnector, appConfig: ApplicationCon
 
   def checkIfJrsClaimsDataExist(nino: Nino)(implicit hc: HeaderCarrier): Future[Boolean] =
     if (appConfig.jrsClaimsEnabled) {
-      jrsConnector.getJrsClaimsForIndividual(nino)(hc).map(_ => true).getOrElse(false)
+      jrsConnector
+        .getJrsClaimsForIndividual(nino)(hc)
+        .map(_.employers.nonEmpty)
+        .getOrElse(false)
```

Comment:
Missing a test for this last case, when getJrsClaimsForIndividual returns None

Reply:
https://github.com/hmrc/tai-frontend/blob/247ef74e4867cc81a15508275f623fd177baf2d8/test/uk/gov/hmrc/tai/service/JrsServiceSpec.scala#L121

We are testing it here